### ISSUE TITLE
Metrics Generator Documentation needs updated around ingestion_time_range_slack

### ISSUE THREAD

**Issue by EStork09** [grafana/tempo#2508]
**Describe the bug**

ingestion_time_range_slack is not a valid configuration option for the metric generator.

**To Reproduce**
1. Start Tempo (SHA or version)
2. set ingestion_time_range_slack in metrics generator config block

**Expected behavior**

Not error out on documented value.

**Environment:**
 - Infrastructure: Kubernetes
 - Deployment tool: helm

**Additional Context**

Should the docs be updated to match metrics_ingestion_time_range_slack or do we want the yaml configuration to match ingestion_time_range_slack? I am assuming metrics_ingestion_time_range_slack is the correct value.
I can make the PR to address it, I mainly am putting this to track it.
[typo](https://github.com/grafana/tempo/blob/main/docs/sources/tempo/configuration/_index.md?plain=1#L355)
[expected value](https://github.com/grafana/tempo/blob/main/modules/generator/config.go#L35)

**Comment by joe-elliott** [comment]
yup, I believe the simplest change would be to just update the config documentation with the correct name. Changing the name is a bit more involved as it's a breaking change. We can handle that in the future if we want to change the name.

> I can make the PR to address it, I mainly am putting this to track it.
If you don't mind submitting a PR to correct the name in the documentation it would be much appreciated :pray: .
